index.js

```diff
@@ -12,46 +12,45 @@ var options = {
 
 module.exports = function(privateKey, remoteIP, response, cb) {
   var error = null;
-  
+
   if (!privateKey) {
     error = 'Private key is required';
-  } else if (!remoteIP) {
```

Comment:
And changing the params in that way to expect an object would be a breaking change, thus a no go in this case.